src/run.py

```diff
@@ -62,50 +68,103 @@ def export(self, export_args: Dict) -> None:
             'era5': ERA5Exporter,
             'vhi': VHIExporter,
             'era5POS': ERA5ExporterPOS,
+            'gleam': GLEAMExporter,
+            'chirps': CHIRPSExporter,
+            'srtm': SRTMExporter
```

Comment:
can we add the KenyaAdminBoundaries exporter too? Or does this need a more involved function?

```
def run_region_exporter(selection):
    to_region = { 
        'kenya' : KenyaAdminExporter ,
        'ethiopia' : EthiopiaAdminExporter ,
        'east_africa' : EastAfricaAdminExporter
    }
```

Maybe I can do this in a separate PR?

Reply:
probably makes most sense to do in a different PR

Reply:
added, then removed.

On reflection, could we update the API for the boundaries exporting class - I think having a unique class for each export seems clunky, especially since (for the two which work, kenya and ethiopia), what happens in the export functions is essentially the same, just with a different url.

We could collapse it into a single class, or (like the engineer) have them be called from an overarching `Boundaries` class.

Reply:
(my main concern is if we expand this to include all the countries, there will be an overwhelming amount of exporters / preprocessors for countries)